src/cmd/top/joblist_pane.c

```diff
@@ -99,6 +99,21 @@ void joblist_pane_draw (struct joblist_pane *joblist)
     wattroff (joblist->win, A_REVERSE);
     if (joblist->jobs == NULL)
         return;
+
+    if (json_array_size (joblist->jobs) == 0
+        && queues_configured (joblist->top->queues)) {
+        const char *filter_queue = NULL;
+        /* can return NULL filter_queue for "all" queues */
+        queues_get_queue_name (joblist->top->queues, &filter_queue);
+        if (filter_queue)
+            mvwprintw (joblist->win,
+                       10,
+                       25,
```

Comment:
Maybe just leave the note at the top? What if the terminal has < 25 rows?

Reply:
yeah, i guess I wanted to center it to be pretty, but perhaps on line 2 or 3 would be preferable.

Reply:
oh wait a second, 25 is the width ... it starts on line 10 ... do you feel that's too far down?

Reply:
Oh, oops. I felt I had a window that wasn't too small, and the text was near the bottom. But I forget how this works, is that line 10 of the pane? I'm fine with keeping it as is, it was just a suggestion.

Reply:
Yeah, line 10 of the pane.  But we can bump it up a bit, maybe line 5 or something.